docs/APIs.md

```diff
@@ -89,7 +89,8 @@ If you want to allow custom formatting, you may provide a custom sanitization fu
 
```

Comment:
I wanted you to update `sanitize: (value) => value.replace(/&/g, '&amp;').replace(/</g, '&lt;')` above ;-)